src/Recruit.Vacancies.Client/Domain/Entities/Vacancy.cs

```diff
@@ -17,6 +17,17 @@ public class Vacancy
 
         public DateTime? SubmittedDate { get; set; }
 
-        public bool CanSubmit => Status == VacancyStatus.Draft;
+        public bool IsDeleted { get; set; }
```

Comment:
Shouldn't IsDeleted be derived from Status?

Reply:
Had a chat to Mark about this this morning. We want to know what status the vacancy was in before it was deleted so using a separate bool field for deletions